README.md

```diff
@@ -39,21 +38,11 @@ Simply add it to the folder containing your component. The browser test command
 
 ## Environment
 
-We haven't yet added any utilities to help load environment variables; you can do it yourself by typing in the following in your terminal:
+This project requires several environment variables be configured before it is able to run. It uses a library called [`habitat`](https://github.com/brianloveswords/habitat) to load configuration from a `.env` file.
 
-e.g.
-```
-export HOST=0.0.0.0
-export PORT=1234
-export LOGINAPI=https://user:password@login.webmaker.org
-export COOKIE_SECRET=topSecretPasswordForEncryptingCookies
-export SECURE_COOKIES=true
-export URI=https://id.webmaker.org
-export GA_TRACKING_ID='UA-000000-01'
-export GA_DEBUG='on'
-export OPTIMIZELY_ID='206878104'
-export OPTIMIZELY_ACTIVE='yes'
-```
+A [`sample.env` file](https://github.com/mozilla/id.webmaker.org/blob/develop/sample.env) is included with this repository. Simply copy this file to a file called `.env` to use the default configuraiton. See the "Up and Running" section above for more information.
```

Comment:
The tiniest of less confusing by rephrasing as "Simply create a copy of this file called `.env`", since copying a file to a file is not something everyone understands. Also, small typo in "configuration"